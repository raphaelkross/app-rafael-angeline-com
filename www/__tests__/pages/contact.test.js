/* eslint-env jest */

import renderer from "react-test-renderer";
import Contact from "../../pages/contact";

describe("Contact", () => {
	it("should render complete layout", () => {
		const component = renderer.create(<Contact />);

		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
