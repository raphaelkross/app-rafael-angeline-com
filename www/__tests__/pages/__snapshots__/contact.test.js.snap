// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact should render complete layout 1`] = `
Array [
  <header
    className="jsx-3590052481"
  >
    <div
      className="jsx-3590052481 container"
    >
      <nav
        className="jsx-3590052481"
      >
        <ul>
          <li>
            <a
              href="/"
              onClick={[Function]}
            >
              <span>
                Home.
              </span>
            </a>
          </li>
          <li>
            <a
              href="/about"
              onClick={[Function]}
            >
              <span>
                About.
              </span>
            </a>
            <ul>
              <li>
                <a
                  href="https://twitter.com/RafaelAngeline"
                  rel="nofollow"
                  target="_blank"
                >
                  <span>
                    Twitter.
                  </span>
                </a>
              </li>
              <li>
                <a
                  href="https://github.com/raphaelkross"
                  rel="nofollow"
                  target="_blank"
                >
                  <span>
                    Github.
                  </span>
                </a>
              </li>
              <li>
                <a
                  href="https://profiles.wordpress.org/rafaelangeline"
                  rel="nofollow"
                  target="_blank"
                >
                  <span>
                    WordPress.
                  </span>
                </a>
              </li>
            </ul>
          </li>
          <li>
            <a
              href="/work"
              onClick={[Function]}
            >
              <span>
                Work.
              </span>
            </a>
          </li>
          <li>
            <a
              href="/contact"
              onClick={[Function]}
            >
              <span>
                Contact.
              </span>
            </a>
          </li>
        </ul>
      </nav>
      <div
        className="jsx-3590052481 responsive-trigger"
      >
        <a
          className="jsx-3590052481 responsive-trigger-link"
          href="#"
          onClick={[Function]}
        >
          <svg
            className="jsx-3590052481 feather feather-menu"
            fill="none"
            height="24"
            stroke="currentColor"
            strokeLinecap="round"
            strokeLinejoin="round"
            strokeWidth="1"
            viewBox="0 0 24 24"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <line
              className="jsx-3590052481"
              x1="3"
              x2="21"
              y1="12"
              y2="12"
            />
            <line
              className="jsx-3590052481"
              x1="3"
              x2="21"
              y1="6"
              y2="6"
            />
            <line
              className="jsx-3590052481"
              x1="3"
              x2="21"
              y1="18"
              y2="18"
            />
          </svg>
        </a>
      </div>
    </div>
    <div
      className="jsx-3590052481 modal"
    >
      <a
        className="jsx-3590052481 modal-close"
        href="#"
        onClick={[Function]}
      >
        <svg
          className="jsx-3590052481 feather feather-x"
          fill="none"
          height="24"
          stroke="currentColor"
          strokeLinecap="round"
          strokeLinejoin="round"
          strokeWidth="1"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <line
            className="jsx-3590052481"
            x1="18"
            x2="6"
            y1="6"
            y2="18"
          />
          <line
            className="jsx-3590052481"
            x1="6"
            x2="18"
            y1="6"
            y2="18"
          />
        </svg>
      </a>
      <div
        className="jsx-3590052481"
      >
        <ul>
          <li>
            <a
              href="/"
              onClick={[Function]}
            >
              <span>
                Home.
              </span>
            </a>
          </li>
          <li>
            <a
              href="/about"
              onClick={[Function]}
            >
              <span>
                About.
              </span>
            </a>
            <ul>
              <li>
                <a
                  href="https://twitter.com/RafaelAngeline"
                  rel="nofollow"
                  target="_blank"
                >
                  <span>
                    Twitter.
                  </span>
                </a>
              </li>
              <li>
                <a
                  href="https://github.com/raphaelkross"
                  rel="nofollow"
                  target="_blank"
                >
                  <span>
                    Github.
                  </span>
                </a>
              </li>
              <li>
                <a
                  href="https://profiles.wordpress.org/rafaelangeline"
                  rel="nofollow"
                  target="_blank"
                >
                  <span>
                    WordPress.
                  </span>
                </a>
              </li>
            </ul>
          </li>
          <li>
            <a
              href="/work"
              onClick={[Function]}
            >
              <span>
                Work.
              </span>
            </a>
          </li>
          <li>
            <a
              href="/contact"
              onClick={[Function]}
            >
              <span>
                Contact.
              </span>
            </a>
          </li>
        </ul>
      </div>
    </div>
  </header>,
  <section
    className="jsx-1057171922 first white"
  >
    <div
      className="jsx-1057171922 container"
    >
      <h1
        className="jsx-1057171922 page-title"
      >
        Get in touch.
      </h1>
      <h1
        className="jsx-3503807385"
      >
        <a
          className="jsx-3503807385"
          href="mailto:me@rafaelangeline.com"
        >
          me@rafaelangeline.com
        </a>
      </h1>
      <p
        className="jsx-3503807385"
      >
        I love talking about projects, opportunities, new tech or whatever you think it's worth a conversation! Don't hesitate to get in touch :)
      </p>
    </div>
  </section>,
  <footer
    className="jsx-7675024"
  >
    <div
      className="jsx-7675024 container"
    >
      <div
        className="jsx-7675024 widget"
      >
        <h3
          className="jsx-7675024"
        >
          Contact.
        </h3>
        <p
          className="jsx-7675024"
        >
          Rafael Angeline
          <br
            className="jsx-7675024"
          />
          Send me your message
          <br
            className="jsx-7675024"
          />
          me [at] rafaelangeline.com
        </p>
      </div>
      <div
        className="jsx-7675024 widget"
      >
        <h3
          className="jsx-7675024"
        >
          Location.
        </h3>
        <p
          className="jsx-7675024"
        >
          Magic Happens at
          <br
            className="jsx-7675024"
          />
          São Paulo, SP - Brazil
        </p>
      </div>
      <div
        className="jsx-7675024 widget"
      >
        <h3
          className="jsx-7675024"
        >
          Stay in Touch.
        </h3>
        <p
          className="jsx-7675024"
        >
          Follow me on
           
          <a
            className="jsx-7675024 twitter"
            href="https://twitter.com/RafaelAngeline"
            rel="me"
            target="_blank"
          >
            <span
              className="jsx-7675024"
            >
              Twitter
            </span>
          </a>
          ,
           
          <a
            className="jsx-7675024 stackoverflow"
            href="http://stackexchange.com/users/8383862/rafael-angeline"
            rel="me nofollow"
            target="_blank"
          >
            <span
              className="jsx-7675024"
            >
              StackOverflow
            </span>
          </a>
          ,
           
          <a
            className="jsx-7675024 github"
            href="https://github.com/raphaelkross"
            rel="me nofollow"
            target="_blank"
          >
            <span
              className="jsx-7675024"
            >
              Github
            </span>
          </a>
          ,
           
          <a
            className="jsx-7675024 wordpress"
            href="https://profiles.wordpress.org/rafaelangeline"
            rel="me nofollow"
            target="_blank"
          >
            <span
              className="jsx-7675024"
            >
              WordPress
            </span>
          </a>
        </p>
      </div>
      <p
        className="jsx-7675024 copy"
      >
        © Copyright Rafael Angeline. Built by Rafael Angeline with React, Next.js and WordPress &lt;3
      </p>
    </div>
  </footer>,
]
`;
